WooCommerce/Classes/Authentication/Prologue/LoginOnboardingPageViewController.swift

```diff
@@ -6,8 +6,6 @@ final class LoginOnboardingPageViewController: UIPageViewController {
 
     private let pages: [UIViewController]
 
-    private let pageControl = UIPageControl()
```

Comment:
Switching to using the built-in page control of the `UIPageViewController` simplifies the code quite a bit, and also improves the UI, as we don't need to apply any hardcoded spacing to the pages themselves (I mean the removal of `stackBottomMargin` below).